### PR TITLE
adding Photos++ 354

### PR DESCRIPTION
**Pull request by inugent**
Request to add Photos++ 3.54 only to CMSSW. This request will be associated with a pull request for the interface in CMSSW.

### CODE REVIEW

File: photospp.spec [hunk 1]
```diff
@@ -0,0 +1,44 @@
+### RPM external photos 3.54
```
Comment (ghost) [inline photospp.spec:1]: The name of the package needs to be consistent: should be photospp.

File: photospp.spec [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+%if "%{?cms_cxx:set}" != "set"
+%define cms_cxx c++
+%endif
+
+%if "%{?cms_cxxflags:set}" != "set"
+%define cms_cxxflags -std=c++0x -g -O2
+%endif
+
+%define keep_archives true
```
Comment (ghost) [inline photospp.spec:17]: Is it really necessary to keep archives?

**Comment by ghost** [comment]
Tested and built. (with some minor tweek)
However it will be discussed in the next ORP whether we should include a new package.

**Comment by inugent** [comment]
Hello,
  The patch file is for a bug reported by the authors (It was provided by the authors of Tauola). We have recently found out that the authors are planning to put forward a new version of Photos fixing the bugs related to gcc 480+. I would request that this pull be put on hold until we hear back from the Photos authors. 
Ian

**Pull request closed without merging**